tools/sidekick/plugins.js

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2022 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+
+const sk = document.querySelector('helix-sidekick');
+sk.addEventListener('custom:generate-email-template', () => {
+  const resp = await fetch(`./export-template.html`);
+  const template = document.createElement('html');
+  template.innerHTML = await resp.text();
+
+  // Make images url's absolute.. Not sure why this works.. else it's relative
```

Comment:
This works because `img.src` is a property on the DOM node which contains a processed version of the value from the HTML tag's `src` attribute. It auto-converts a relative to an absolute URL, like `a.href` vs `a.getAttribute('href')`.

If you want to make the assignment below more explicit, you could write
```js
img.setAttribute('src', new URL(img.getAttribute('src'), window.location.origin));
```
to achieve the same result...